workers/unity/Packages/com.improbable.gdk.core/Worker/SpatialOSWorld.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class SpatialOSWorld : World
+    {
+        private WorkerBase worker;
+        private List<ScriptBehaviourManager> spatialOSSystemManagers;
```

Comment:
Really annoying the equivalent of this is private on the base `World` class

Reply:
I know :(